src/model/storage-strategy.js

```diff
@@ -5,7 +5,8 @@
 const StorageStrategy = {
   cookie: 'cookie',
   localStorage: 'ls',
-  none: 'none'
+  none: 'none',
+  disabled: 'disabled'
```

Comment:
We should describe it in [CONFIGURATION_OPTIONS](https://github.com/LiveIntent/live-connect/blob/master/CONFIGURATION_OPTIONS.md) and also mention that  it will the the behaviour in case gdprApplies is set to true.